locustfile.py

```diff
@@ -51,47 +436,26 @@ def login(self):
             if post_response.url != HOST + redirect_url:
                 post_response.failure('Authentication failed')
 
-    @task(1)
-    def index(self):
-        self.client.get('/')
+    # Create a plan and store the ID on self
+    def create_plan(self):
+        get_response = self.client.get('/districtmapping/plan/0/view/')
+        post_response = self.client.post(
+            '/districtmapping/plan/{template}/copy/'.format(template=TEMPLATE_PLAN_ID),
+            {'legislativeBody': LEGISLATIVE_BODY_ID,
+             'name': ''.join([random.choice(string.ascii_uppercase) for _ in range(8)])},
+            headers={'X-CSRFToken': get_response.cookies['csrftoken']},
+            catch_response=True
+        )
+        self.plan_id = post_response.json()[0]['pk']
 
-    @task(2)
-    def plan(self):
-        self.client.get(
-            '/districtmapping/plan/{plan_id}/edit/'.format(plan_id=PLAN_ID))
-
-    @task(10)
-    def reassign(self):
+    def get_edit_csrf(self):
         get_response = self.client.get(
```

Comment:
It doesn't look like we're grouping `edit`, `view`, or `demographics` requests. I think we should since it makes the stats more readable.